src/main/java/com/java_essential/kopaiev/essential/homework/lesson1/task4/Computer.java

```diff
@@ -0,0 +1,12 @@
+package com.java_essential.kopaiev.essential.homework.lesson1.task4;
+
+public class Computer {
+    public static void main(String[] args) {
+
+        Object[] computers = new Object[5];
+
+        for (int i = 0; i < computers.length; i++) {
+            computers[i] = new Computer();
+        }
```

Comment:
Create five instance of Computer by HANDS, than, with loop, add them to the list)))

Reply:
Probably fixed. Changed according to approved Pull Request